tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -1444,3 +1444,249 @@ def test_invalid_list_recommendations_notification_codes(cluster_type: str):
+
+                # Get the experiment name
+                json_data = json.load(open(create_exp_json_file))
+                experiment_name = json_data[0]['experiment_name']
+
+                response = list_recommendations(experiment_name)
+                assert response.status_code == SUCCESS_200_STATUS_CODE
+
+                recommendation_json = response.json()
+
+                recommendation_section = None
+
+                for containers in recommendation_json[0]["kubernetes_objects"][0]["containers"]:
+                    actual_container_name = containers["container_name"]
+                    print(
+                        f"actual container name = {actual_container_name}  expected container name = {container_name_to_update}")
+                    if containers["container_name"] == container_name_to_update:
+                        recommendation_section = containers["recommendations"]
+                        break
+
+                assert recommendation_section is not None
+
+                high_level_notifications = recommendation_section["notifications"]
+
+                # Check for Recommendation level notifications
+                assert INFO_RECOMMENDATIONS_AVAILABLE_CODE in high_level_notifications
+
+                data_section = recommendation_section["data"]
+                # Check if recommendation exists
+                assert str(end_time) in data_section
+                # Check for timestamp level notifications
+                timestamp_level_notifications = data_section[str(end_time)]["notifications"]
+                assert INFO_SHORT_TERM_RECOMMENDATIONS_AVAILABLE_CODE in timestamp_level_notifications
+
+                # Check for current recommendation
+                recommendation_current = None
+                if "current" in data_section[str(end_time)]:
+                    recommendation_current = data_section[str(end_time)]["current"]
+
+                short_term_recommendation = data_section[str(end_time)]["recommendation_terms"]["short_term"]
+                if j > 671: # 7 days
+                    medium_term_recommendation = data_section[str(end_time)]["recommendation_terms"]["medium_term"]
+                if j > 1439: # 15 days
+                    long_term_recommendation = data_section[str(end_time)]["recommendation_terms"]["long_term"]
+
+                if INFO_COST_RECOMMENDATIONS_AVAILABLE_CODE in short_term_recommendation["notifications"]:
+                    assert "variation" in short_term_recommendation["recommendation_engines"]["cost"]
+                    assert "config" in short_term_recommendation["recommendation_engines"]["cost"]
+
+                if INFO_PERFORMANCE_RECOMMENDATIONS_AVAILABLE_CODE in short_term_recommendation["notifications"]:
+                    assert "variation" in short_term_recommendation["recommendation_engines"]["performance"]
+                    assert "config" in short_term_recommendation["recommendation_engines"]["performance"]
```

Comment:
We need to validate the cpu or memory values here for all the terms

Reply:
@chandrams added the value validation based on current config as both should be same